components/sandbox/toolchain-host-operator/production/stone-prd-host1/miner-detector-cronjob.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: batch/v1
```

Comment:
You need to include this file in the kustomization.yaml

Reply:
🤦‍♂️ as always 

thanks @hugares done in https://github.com/redhat-appstudio/infra-deployments/pull/3321/commits/b77355dbdeb11ecd550a6c23f0d18e78561e1524